mkt/websites/management/commands/import_games_from_csv.py

```diff
@@ -0,0 +1,138 @@
+# docs.google.com/spreadsheets/d/
+# 1tShUCIR3tilh2WGeoXQMjy4ExGf94_7MilZqorjPfrM/edit#gid=0
+import csv
+from os.path import basename, splitext
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.validators import URLValidator, ValidationError
+from django.db.transaction import atomic
+
+from mkt.constants.applications import DEVICE_DESKTOP
+from mkt.constants.base import STATUS_PUBLIC
+from mkt.tags.models import Tag
+from mkt.websites.models import Website
+from mkt.websites.tasks import fetch_icon, fetch_promo_imgs
+
+
+class ParsingError(Exception):
+    pass
+
+
+class Command(BaseCommand):
+    """
+    Usage:
+
+        python manage.py import_games_from_csv <file>
+
+    Batch import games from a CSV. If there is a validation error on anything,
```

Comment:
Haven't done this yet...might be worth doing, but I will possibly forego this since we have a go/no-go tomorrow.